src/libraries/ErrorsLib.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {Id} from "../../lib/metamorpho/src/interfaces/IMetaMorpho.sol";
+
+/// @title ErrorsLib
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Library exposing error messages.
+library ErrorsLib {
+    /// @notice Thrown when the `msg.sender` is not the `owner`.
+    error NotOwner();
+
+    /// @notice Thrown when the address passed is the zero address.
+    error ZeroAddress();
+
+    /// @notice Thrown when allocation to market `id` exceeds current max inflow
+    error InflowCapExceeded(Id id);
+
+    /// @notice Thrown when allocation from market `id` exceeds current max outflow
+    error OutflowCapExceeded(Id id);
+
+    /// @notice Thrown when flow configuration for market `id` has min flow > max flow
+    error InconsistentFlowConfig(Id id);
+
+    /// @notice Thrown when the reallocation fee given is wrong
+    error IncorrectFee(uint givenFee);
```

Comment:
I don't think that it's useful (you know how much you where paying no?)
```suggestion
    error IncorrectFee();
```

Reply:
34cac9fb9e4994ae651490c4c09a8973e1051711